projects/RPi2/filesystem/usr/lib/systemd/system/serial-console.service

```diff
@@ -5,7 +5,8 @@ ConditionKernelCommandLine=console
 
 [Service]
 WorkingDirectory=/storage
-ExecStart=/bin/sh -c '. /etc/profile; exec /bin/sh'
+Environment="ENV=/etc/profile"
+ExecStart=/bin/sh -c 'exec /bin/sh'
```

Comment:
I don't have any aliases so I haven't noticed the issue you are describing. A quick check showed that aliases indeed aren't inherited. I also don't see why you need the exec in this case.